analysis/train_value_function.py

```diff
@@ -30,9 +29,12 @@ def train_value_function(
                 cache_directory=world.get_train_directory(save_suffix), suffix=shift
             )
 
+        # avoid running the world after the last model is saved
         if shift != number_of_shifts:
-            # avoid running the world after the last model is saved
-            policy_world.run()
+            if scenario_training:
+                scenario_simulation.scripts.training_simulation(policy_world)
```

Comment:
Maybe a bit strange to call it scenario simulation when it is not?

Reply:
Agree, call it traning_simulation?